linera-faucet/server/src/lib.rs

```diff
@@ -25,6 +25,9 @@ use serde::Deserialize;
 use tower_http::cors::CorsLayer;
 use tracing::info;
 
+/// A rough estimate of the maximum fee for a block.
+const MAX_FEE: Amount = Amount::from_millis(100);
```

Comment:
This won't be necessary after https://github.com/linera-io/linera-protocol/pull/3850 ?

Reply:
There will still be some fees left, because we are making an operation (rather than processing a message with a grant).

Reply:
Ah, OK. I thought it's the fee for block creation only